src/schoology-extractor/schoology_extractor/grading_periods.py

```diff
@@ -25,21 +25,26 @@
         logging.StreamHandler(sys.stdout),
     ],
     format=logFormatter,
-    level=log_level
+    level=log_level,
 )
 logger = logging.getLogger(__name__)
 
 logger.info("Ed-Fi LMS Schoology Extractor - Grading Periods")
 request_client = RequestClient(schoology_key, schoology_secret)
 
 logger.info("Getting grading periods")
+
 try:
+
     grading_periods_list: List[Any] = []
     grading_periods_response = request_client.get_grading_periods()
     while True:
```

Comment:
Try to execute as few lines as possible in try/catch blocks. Initializing a variable should not fail.
```suggestion
    grading_periods_list: List[Any] = []

try:

    grading_periods_response = request_client.get_grading_periods()
    while True:
```

Reply:
While I understand the sentiment, there is no functional difference so I don't see any value to changing the code here.